server/store_statistics.go

```diff
@@ -120,6 +120,44 @@ func (s *storeStatistics) Collect() {
 		clusterStatusGauge.WithLabelValues(typ, s.namespace).Set(value)
 	}
 
+	// Current scheduling configurations of the cluster
+	configs := make(map[string]float64)
+	configs["leader_schedule_limit"] = float64(s.opt.GetLeaderScheduleLimit(s.namespace))
+	configs["region_schedule_limit"] = float64(s.opt.GetRegionScheduleLimit(s.namespace))
+	configs["merge_schedule_limit"] = float64(s.opt.GetMergeScheduleLimit(s.namespace))
+	configs["replica_schedule_limit"] = float64(s.opt.GetReplicaScheduleLimit(s.namespace))
+	configs["max_replicas"] = float64(s.opt.GetMaxReplicas(s.namespace))
+	configs["high_space_ratio"] = float64(s.opt.GetHighSpaceRatio())
+	configs["low_space_ratio"] = float64(s.opt.GetLowSpaceRatio())
+	configs["tolerant_size_ratio"] = float64(s.opt.GetTolerantSizeRatio())
+
+	var disableMakeUpReplica, disableLearner, disableRemoveDownReplica, disableRemoveExtraReplica, disableReplaceOfflineReplica float64
+	if !s.opt.IsMakeUpReplicaEnabled() {
+		disableMakeUpReplica = 1
+	}
+	if !s.opt.IsRaftLearnerEnabled() {
+		disableLearner = 1
+	}
+	if !s.opt.IsRemoveDownReplicaEnabled() {
+		disableRemoveDownReplica = 1
+	}
+	if !s.opt.IsRemoveExtraReplicaEnabled() {
+		disableRemoveExtraReplica = 1
+	}
+	if !s.opt.IsReplaceOfflineReplicaEnabled() {
+		disableReplaceOfflineReplica = 1
+	}
+
+	configs["disable_makeup_replica"] = disableMakeUpReplica
```

Comment:
Can we just do `configs["disable_makeup_replica"] = !s.opt.IsMakeUpReplicaEnabled()`?